pkg/config/config.go

```diff
@@ -22,10 +22,16 @@ import (
 	"github.com/runfinch/finch/pkg/system"
 )
 
+// AdditionalDirectory represents the additional directory used in Finch config.
```

Comment:
We can probably refine the comment according to the description of it in README as it may be a bit too vague now.

Reply:
AdditionalDirectory is just the element of the whole AdditionalDirectories instead of serving the complicated use case by itself. Does it really make sense of adding the use case of AdditionalDirectories here?

Reply:
Then maybe at least add some comments to `AdditionalDirectories`? Currently looking at the field definitions basically give no info what they are used for, while we shouldn't need to trace their references only to get a high-level understanding of them.